tests/unit/acquisition/test_optimizer.py

```diff
@@ -54,44 +59,30 @@ def test_optimize_discrete(
     ],
 )
 def test_optimize_continuous(
-    search_space: Box,
-    shift: list[float],
-    expected_maximizer: list[list[float]],
+    search_space: Box, shift: list[float], expected_maximizer: list[list[float]],
 ) -> None:
-    maximizer = optimize_continuous(search_space, lambda x: 0.5 - quadratic(x[0, :] - shift))
+    maximizer = optimize_continuous(search_space, _quadratic_sum(shift))
     npt.assert_allclose(maximizer, expected_maximizer, rtol=2e-4)
 
 
+def _branin_sum(x: TensorType) -> TensorType:
+    return -tf.reduce_sum(branin(x), axis=-2)
+
+
 @random_seed
-@pytest.mark.parametrize(
-    "search_space, shifts, batch_size, expected_maximizer",
-    [
-        (Box([-1], [2]), [1.0, 1.5], 2, [[1.0], [1.5]]),  # 1D space with batch of size 2
-        (
-            Box([-1, -2], [1.5, 2.5]),
-            [[0.3, -0.4]] * 3,
-            3,
-            [[0.3, -0.4]] * 3,
-        ),  # 2D space with batch of size 3
-        (
-            Box([-1, -2, 1], [1.5, 2.5, 1.5]),
-            [[0.3, -0.4, 0.5]] * 5,
-            5,
-            [[0.3, -0.4, 1.0]] * 5,
-        ),  # 3D space with a batch of size 5
-    ],
-)
-def test_optimize_continuous_batch(
-    search_space: Box,
-    shifts: list[float],
-    batch_size: int,
-    expected_maximizer: list[list[float]],
+@pytest.mark.parametrize("batch_size", [1, 2, 3, 5])
+@pytest.mark.parametrize("search_space, acquisition, maximizers", [
+    (Box([0], [1]), _quadratic_sum([0.5]), ([[0.5, -0.5]])),
+    (Box([0, 0], [1, 1]), _branin_sum, BRANIN_MINIMIZERS),
+    (Box([0, 0, 0], [1, 1, 1]), _quadratic_sum([0.5, -0.5, 0.2]), ([[0.5, -0.5, 0.2]])),
+])
+def test_batchify(
```

Comment:
this test isn't necessarily preferable to yours - i just found it easier to start from scratch